### DISCUSSION TITLE
菜单栏（Menubar）+ 桌面客户端

### DISCUSSION THREAD

**Discussion by rainyear** [flomoapp/3rd-party-tools#42] [分享创造]
基于 Electron + [Menubar](https://github.com/maxogden/menubar)

#### 或者，你更喜欢快捷键呼出（`Shift + Cmd + F`）

![FlomoBar_Enter](https://user-images.githubusercontent.com/3953966/115993569-03480c00-a606-11eb-97d2-4c97fadfcb8f.png)

#### 叮咚！从 `FlomoBar` 成功添加一条记录
![Notification](https://user-images.githubusercontent.com/3953966/115993594-1c50bd00-a606-11eb-9154-46f897065199.png)

#### 摆脱浏览器，即刻进入 Flomo 桌面端！
![Desktop](https://user-images.githubusercontent.com/3953966/115993608-283c7f00-a606-11eb-9de5-33593321849c.png)

Windows 版本（快捷键：`Win+Shift+F`）：
![SharedScreenshot](https://user-images.githubusercontent.com/3953966/115742649-79a8fc00-a3c3-11eb-9d1b-cd75b6d92e69.jpg)

> Windows 版本桌面客户端正在打包中…

**下载地址：** https://github.com/rainyear/FlomoBar/releases/tag/2.0

**Comment by plidezus** [comment]
牛批！！！

**Reply by vcfger** [reply]
👍

**Comment by Seulxy** [comment]
小小催更一下……等着windows版本！
